docs/static/settings/pipeline-config-settings.asciidoc

```diff
@@ -0,0 +1,27 @@
+// The content for these settings is single sourced from attributes-ls.asciidoc
+
+{path-config}:: {path-config-desc} 
+
+{config-string}:: {config-string-desc} 
+
+{config-test_and_exit}:: {config-test_and_exit-desc} 
+
+{config-reload-automatic}:: {config-reload-automatic-desc} 
+
+{config-reload-interval}:: {config-reload-interval-desc} 
+
+{config-debug}:: {config-debug-desc}
+
+{config-support_escapes}:: {config-support_escapes-desc}
+
+{config-field-ref-escape-style}:: {config-field-ref-escape-style-desc}
+
+IMPORTANT: The following `config.field_reference.escape_style` setting is a duplicate of the previous setting while we resolve formatting issues. 
+
+config.field_reference.escape_style:: [Technical preview]. Provides a way to reference fields that contain <<formal-grammar-escape-sequences,field reference special characters>> `[` and `]`. +
+Current options are:   +
+* `percent`: URI-style `%`+`HH` hexadecimal encoding of UTF-8 bytes (`[` -> `%5B`; `]` -> `%5D`). +
+* `ampersand`: HTML-style `&#`+`DD`+`;` encoding of decimal Unicode code-points (`[` -> `&#91;`; `]` -> `&#93;`). +
+* `none`: field names containing special characters _cannot_ be referenced. 
++
+NOTE: This feature is in *technical preview* and may change in the future.
```

Comment:
@lcawl, here's a section that highlights some of the formatting limitations I'm bumping into with using attributes in a definition list.  Check out `config.field_reference.escape_style` for a good example. 

https://logstash_14210.docs-preview.app.elstc.co/guide/en/logstash/master/logstash-yml-settings.html#settings-pipeline-config

Reply:
The [Asciidoctor docs](https://docs.asciidoctor.org/asciidoc/latest/attributes/document-attributes/#what-does-defining-a-document-attribute-mean) seem to say that attributes work if you "have a value that includes basic inline AsciiDoc syntax, such as: ...
text formatting (if wrapped in a [pass macro](https://docs.asciidoctor.org/asciidoc/latest/pass/pass-macro/#inline-pass))...". However, I did some quick tests with `pass:q[formatted values]` and it didn't seem to work. We might not have that functionality enabled in our doc builds. I did, however, get the formatting to work by using tagged regions. If you're interested in that method, see https://github.com/karenzone/logstash/pull/1